stumpy/scrump.py

```diff
@@ -186,31 +250,48 @@ def _compute_diagonal(
 
 
 @njit(parallel=True, fastmath=True)
-def _scrump(T, m, μ, σ, orders, orders_ranges, percentage=0.1):
+def _scrump(T_A, T_B, m, M_T, Σ_T, μ_Q, σ_Q, orders, orders_ranges, ignore_trivial):
     """
     A Numba JIT-compiled version of SCRIMP (self-join) for parallel computation
     of the matrix profile and matrix profile indices.
 
     Parameters
     ----------
-    T : ndarray
+    T_A : ndarray
         The time series or sequence for which to compute the matrix profile
 
+    T_B : ndarray
+        The time series or sequence that contain your query subsequences
+        of interest
+
     m : int
         Window size
 
-    μ : ndarray
-        Sliding window mean for T
+    M_T : ndarray
+        Sliding mean of time series, `T`
 
-    σ : ndarray
-        Sliding window standard deviation for T
+    Σ_T : ndarray
+        Sliding standard deviation of time series, `T`μ_Q : ndarray
+        Mean of the query sequence, `Q`, relative to the current sliding window
```

Comment:
Remove "μ_Q : ndarray Mean of the query sequence, `Q`, relative to the current sliding window" as it is redundant from next line